students/ScottL/session04/Trigrams.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+
+
+def word_list_generator(file_location):
+    """
+    Return a sequence of words, stripped of punctuation & capitalization, taken from a source file.
+    :param file_location: a relative or absolute address to use as source of plain text
+    :return: a sequence of individual or compound words
+    """
+    with open(file_location) as file:
+        original_words = file.read().split()
+    edited_words = []
+    replaced_chars = ""
+    replacement_chars = ""
+    deleted_chars = ".,[]();'!?:<>*^"
```

Comment:
you can `import string` and do `string.punctuation()` to get all the possible values to remove.